README.md

```diff
@@ -46,4 +46,4 @@ For example, you can follow [these instructions](https://gorails.com/setup/ubunt
 
 ## View the live app
```

Comment:
Yep, also im Wesentlichen passt die Readme so für mich.
Es sollte grob beschrieben werden, was die App macht und wie ich sie in meiner Entwicklungsumgebung zum Laufen bringe. Ich habe es nicht ausprobiert, aber ich denke das passt schon.
Wir beschreiben außerdem noch bei unseren Projekten, wie wir deployen. Da wir für jede App ein eigenes Ansible-projekt haben, ist das dann ein einfacher Hinweis darauf.

Reply:
Yay! Danke! Soll ich jetzt das PR closen? Mit diese ganzen Datein, die irgenwie auch da sind, vielleicht ist das sogar die einfachste Weg. Was denkst du?